src/usr/local/www/vpn_ipsec_settings.php

```diff
@@ -93,6 +94,10 @@
 		}
 	}
 
+	if (isset($_POST['port_nat_t']) && $_POST['port_nat_t'] == '500' ) {
```

Comment:
This should test using `is_port()` to validate that what was submitted is a valid port value.